wgpu-hal/src/gles/egl.rs

```diff
@@ -145,6 +145,11 @@ struct DisplayOwner {
     display: DisplayRef,
 }
 
+/// # Safety
+/// `library` implements `Send` and `display` is just an enum which gets passed
+/// around.
+unsafe impl Send for DisplayOwner {}
```

Comment:
But is it safe to run the `Drop` code on arbitrary threads? That's what matters here, not just the field contents. In any case, lint fixes shouldn't require adding new impls — see my comment below.

Reply:
oh damn. That's a good point! It looks like that [it's not documented](https://www.x.org/releases/X11R7.6/doc/libX11/specs/libX11/libX11.html#Closing_the_Display) what happens, if you use the same pointer and call `XCloseDisplay` on it multiple times. I'll remove the `Send` implementation. Thank you.

Reply:
“Multiple times” isn't relevant — `Drop` is always executed at most once. The relevant question, to understand whether it's OK to implement `Send` is whether it's okay for `Drop` — or other operations on the type — to be run *on a different thread* than the object was created on.